subprojects/build-cache/src/main/java/org/gradle/caching/local/internal/DirectoryBuildCacheService.java

```diff
@@ -66,9 +69,6 @@ private LoadAction(BuildCacheEntryReader reader) {
         @Override
         public void execute(@Nonnull File file) {
             try {
-                // Mark as recently used
-                GFileUtils.touchExisting(file);
-
```

Comment:
I've removed this because this action is only ever executed in `loadInsideLock` which already touches the file.

@wolfs Can you please double-check?